app/presenters/document_export_presenter.rb

```diff
@@ -27,6 +27,7 @@ def present_edition(edition)
       edition_policies: slice_association(edition, :edition_policies, %i[id policy_content_id]),
       fact_check_requests: present_fact_check_requests(edition),
       government: edition.government&.as_json,
+      history: present_history(edition),
```

Comment:
I wonder if the name `history` could cause confusion with history mode?  Would something like `change_log` or `revision_history` be clearer?

Reply:
Have now changed this to `revision_history` and updated the method name and commit message.